packages/gestalt/src/HelpButton.js

```diff
@@ -0,0 +1,190 @@
+  accessibilityPopoverLabel,
+  idealDirection = 'down',
+  link,
+  text,
+  zIndex,
+}: Props): Node {
+  const ref = useRef(null);
+  const [isOpen, setIsOpen] = useState(false);
+  const [isHover, setIsHover] = useState(false);
+  const popoverId = useId();
+  const {
+    accessibilityTooltipMessage,
+    accessibilityLinkLabel: accessibilityDefaultLinkLabel,
+    accessibilityIcon,
+  } = useDefaultLabelContext('HelpButton');
+
+  useEffect(() => {
+    if (isOpen && ref.current) {
+      ref.current.focus();
+    }
+  }, [isOpen]);
+
+  const toggleView = () => {
+    setIsOpen((currVal) => !currVal);
+  };
+
+  const bgIconColor = isHover ? 'default' : 'subtle';
+
+  const tooltipZIndex = zIndex ?? new FixedZIndex(DEFAULT_ZINDEX - 1);
+
+  return (
+    <Fragment>
+      <Box
+        dangerouslySetInlineStyle={{
+          __style: {
+            // The component main proposal is support or provide more infos to short texts on app
+            // because that the display is inline-block, to better fit with text blocks.
+            display: 'inline-block',
+          },
+        }}
+        tabIndex={-1}
+      >
+        <TapArea
+          accessibilityExpanded={isOpen}
+          accessibilityControls={popoverId}
+          accessibilityLabel={accessibilityTooltipMessage}
+          fullWidth={false}
+          onTap={toggleView}
+          ref={ref}
+          role="button"
+          rounding="circle"
+          onMouseEnter={() => setIsHover(true)}
+          onMouseLeave={() => setIsHover(false)}
+          onFocus={() => setIsHover(true)}
+          onBlur={() => setIsHover(false)}
+        >
+          <Tooltip
+            text={accessibilityTooltipMessage}
+            idealDirection={idealDirection}
+            zIndex={tooltipZIndex}
+          >
+            <Box
+              width={16}
+              height={16}
+              rounding="circle"
+              display="flex"
+              alignItems="center"
+              justifyContent="center"
+              dangerouslySetInlineStyle={{
+                __style: {
+                  backgroundColor: `var(--color-text-${bgIconColor})`,
+                },
+              }}
+            >
+              <Icon
+                accessibilityLabel={accessibilityIcon}
+                color="inverse"
+                icon="question-mark"
+                size={8}
+              />
+            </Box>
+          </Tooltip>
+        </TapArea>
+      </Box>
+      {isOpen && (
+        <Layer
+          zIndex={
+            zIndex
+              ? new CompositeZIndex([new FixedZIndex(zIndex.index())])
+              : new FixedZIndex(DEFAULT_ZINDEX)
+          }
+        >
+          <Popover
+            id={popoverId}
+            accessibilityLabel={accessibilityPopoverLabel}
+            anchor={ref.current}
+            onDismiss={toggleView}
+            idealDirection={idealDirection ?? 'down'}
+            positionRelativeToAnchor={false}
```

Comment:
do we want to leave the door open for potential HelperButtons in fixedContainers? positionRelativeToAnchor 

this is more like a philosophical discussion as many of our POpover based componenrs dont... except Dropdown


this should not be an issue when new POpover is implemented so we can add if we get the request